cardinal/server/handler/tx.go

```diff
@@ -1,48 +1,24 @@
 package handler
 
 import (
-	"errors"
-	"fmt"
-	"time"
-
-	"github.com/coocood/freecache"
-	"github.com/ethereum/go-ethereum/common"
 	"github.com/gofiber/fiber/v2"
 	"github.com/gofiber/fiber/v2/log"
-	"github.com/rotisserie/eris"
 
 	personaMsg "pkg.world.dev/world-engine/cardinal/persona/msg"
 	servertypes "pkg.world.dev/world-engine/cardinal/server/types"
+	"pkg.world.dev/world-engine/cardinal/server/validator"
 	"pkg.world.dev/world-engine/cardinal/types"
 	"pkg.world.dev/world-engine/sign"
 )
 
-const cacheRetentionExtraSeconds = 10 // this is how many seconds past normal expiration a hash is left in the cache.
-// we want to ensure it's long enough that any message that's not expired but
-// still has its hash in the cache for replay protection. Setting it too long
-// would cause the cache to be bigger than necessary
-
-var (
-	ErrNoPersonaTag               = errors.New("persona tag is required")
-	ErrWrongNamespace             = errors.New("incorrect namespace")
-	ErrSystemTransactionRequired  = errors.New("system transaction required")
-	ErrSystemTransactionForbidden = errors.New("system transaction forbidden")
-)
-
 // PostTransactionResponse is the HTTP response for a successful transaction submission
 type PostTransactionResponse struct {
 	TxHash string
 	Tick   uint64
 }
 
-type SignatureVerification struct {
-	IsDisabled               bool
-	MessageExpirationSeconds int
-	HashCacheSizeKB          int
-	Cache                    *freecache.Cache
-}
-
 type Transaction = sign.Transaction
+type SignatureValidator = validator.SignatureValidator
```

Comment:
I generally prefer to not do type alias like this; it makes it less obvious where a certain object/struct is imported from.